internal/testutils/testutils.go

```diff
@@ -69,18 +77,26 @@ func ParseTransportType(s string) (TransportType, error) {
 		return TransportTypeHTTP, nil
 	case "tchannel":
 		return TransportTypeTChannel, nil
+	case "grpc":
+		return TransportTypeGRPC, nil
 	default:
 		return 0, fmt.Errorf("invalid TransportType: %s", s)
 	}
 }
 
-// WithClientConfig wraps a function by setting up a client and server dispatcher and giving
+// ClientInfo holds the client info for testing.
+type ClientInfo struct {
+	ClientConfig   transport.ClientConfig
+	GRPCClientConn *ggrpc.ClientConn
```

Comment:
Note I've filed https://github.com/yarpc/yarpc-go/issues/917 to generally track `grpc.ClientConn` issues.